html/janus.js

```diff
@@ -2386,15 +2380,15 @@ function Janus(gatewayCallbacks) {
 					}
 					// We're going to try and use the extension for Chrome 34+, the old approach
 					// for older versions of Chrome, or the experimental support in Firefox 33+
-					function callbackUserMedia (error, stream) {
+					function callbackUserMedia (error, stream) { // eslint-disable-line no-inner-declarations
```

Comment:
I don't like these comments unrelated to the code lingering here. Is there any alternative?

Reply:
@davel so will you change this or not ?

Reply:
@atoppi I think I already have, you may need to refresh? I changed it into a closure as mentioned.

Reply:
you're right I'm looking at the outdated code, sorry!